src/khepri_fun.erl

```diff
@@ -1132,10 +1168,9 @@ pass2_process_instruction(
   {loop_rec, _, _} = Instruction, State) ->
     replace_label(Instruction, 2, State);
 pass2_process_instruction(
-  {Select, _, _, {list, Cases}} = Instruction,
+  {select_val, _, _, {list, Cases}} = Instruction,
   #state{mfa_in_progress = {Module, _, _},
-         label_map = LabelMap} = State)
-  when Select =:= select_val orelse Select =:= select_tuple_arity ->
+         label_map = LabelMap} = State) ->
```

Comment:
This change looks unrelated to the patch. Was it expected?

Reply:
Whoops, good catch. Git fighting with the editor :upside_down_face: